app/helpers/firm_helper.rb

```diff
@@ -0,0 +1,8 @@
+module FirmHelper
+  def type_of_advice_list_item(firm, type)
+    return unless firm.send(type) == 100
```

Comment:
Is this that this that 100 or 0 should be a boolean thing? Is there anyway we can encapsulate it?